docs-guides/source/mlmodel-utilities.md

```diff
@@ -98,3 +98,60 @@ input_data = [{'input_1': 1, 'input_2': 2}, {'input_1': 3, 'input_2': 3}]
+
+
+### Example
+
+The following code loads the `MobileNetV2.mlpackage` converted in [Getting Started](introductory-quickstart) (from the previously trained [MobileNetV2](https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet_v2) model, which is based on the [tensorflow.keras.applications](https://www.tensorflow.org/api_docs/python/tf/keras/applications)). It uses [`get_weights_metadata()`](https://apple.github.io/coremltools/source/coremltools.optimize.coreml.post_training_quantization.html#coremltools.optimize.coreml.get_weights_metadata), which specifies the size threshold (`2048`) in the `weight_threshold` parameter. A weight tensor is included in the resulting dictionary only if its total number of elements are greater than `weight_threshold`:
+
+```python
+from coremltools.models import MLModel
+from coremltools.optimize.coreml import get_weights_metadata
+
+mlmodel = MLModel("MobileNetV2.mlpackage")
+weight_metadata_dict = get_weights_metadata(mlmodel, weight_threshold=2048)
+
+# iterate over all the weights returned 
+
+large_weights = [] # get the weight names with size > 25600
+sparse_weights = [] # get the weight names with sparsity >= 50% 
+palettized_weights = [] # get the weight names with unique elements <= 16 
+
+for weight_name, weight_metadata in weight_metadata_dict.items():
+     # weight_metadata.val: numpy array of the weight data
+     if weight_metadata.val.size >= 25600: large_weights.append(weight_name)
+
+     # weight_metadata.sparsity: ratio of 0s in the weight, between [0,1]
+     if weight_metadata.sparsity >= 0.5: sparse_weights.append(k)
+
+     # weight_metadata.unique_values : number of unique values in the weight
+     if weight_metadata.unique_values <= 16: palettized_weights.append(k)
+
+     # Access to the child ops this weight feeds into.  
+     child_op = weight_metadata.child_ops[0]
+     # child_op.name: name of the child op
+     # child_op.op_type:  op type of the child op
+     # child_op.params_name_mapping:  this dictionary shows the input parameters of the child op and their corresponding ops' names
```

Comment:
In the doc string, we are mentioning that this new API is mainly for helping when constructing OptimizationConfig .
Even though we do link to the config API doc, do you think in the example, we can also show a super simple use case?
For instance, palettizing the first weight in the model, etc?